### PR TITLE
Reputation colors

### PR DESCRIPTION
**Pull request by g4s8**
#501 - reputation colors legend

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#732](https://github.com/zerocracy/farm/issues/732) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/642009760). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by carlosmiranda** [comment]
@yegor256 good to merge

File: src/main/xsl/team.xsl [hunk 1]
```diff
@@ -67,7 +67,21 @@ SOFTWARE.
         <xsl:text>+</xsl:text>
         <xsl:value-of select="format-number(sum(user/awards) div count(user) div 90,'0')"/>
       </strong>
-      <xsl:text> per user daily).</xsl:text>
+      <xsl:text> per user daily). </xsl:text>
+      <a href="http://www.zerocracy.com/policy.html#18">Reputation</a>
```
Comment (yegor256) [inline src/main/xsl/team.xsl]: @g4s8 don't forget `xsl:text` for all texts

File: src/main/xsl/team.xsl [hunk 2]
```diff
@@ -67,7 +67,21 @@ SOFTWARE.
...
+      <xsl:text> legend: </xsl:text>
+      <span style="color:darkgreen">
+        <xsl:text>trusted (more than 256) </xsl:text>
+      </span>
```
Comment (yegor256) [inline src/main/xsl/team.xsl]: @g4s8 maybe some commas between them?)

**Comment by yegor256** [comment]
@g4s8 it will look weird on the screen( see above

**Comment by g4s8** [comment]
@yegor256 fixed, see new commits

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#732](https://github.com/zerocracy/farm/issues/732) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

No issues were found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka yes, confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 points just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**